libs/wire-api/src/Wire/API/UserEvent.hs

```diff
@@ -212,24 +212,6 @@ managedByUpdate :: UserId -> ManagedBy -> UserEvent
 managedByUpdate u mb =
   UserUpdated $ (emptyUserUpdatedData u) {eupManagedBy = Just mb}
 
-supportedProtocolUpdate :: UserId -> Set BaseProtocolTag -> UserEvent
```

Comment:
you sure about these?  it looks like we may want to actually send these events instead of deleting them?